### ISSUE TITLE
→ Accessibility: Components

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#557]
- [x] El primer paso que vamos a hacer es una auditoría de los elementos existentes en Mística
https://gist.github.com/aweell/9b1f86e398b4fd8706257258fd4df801

## Tasks

- [x] #1487 
- [x] https://github.com/Telefonica/mistica-design/issues/1794
- [x] #1766
- [x] #1489 
- [x] #1490 
- [x] #1492 
- [x] #1800 
- [x] #1759
- [x] #1760
- [x] #1795
- [x] #1801
- [x] #1827
- [x] #2053
- [ ] #1493 
- [ ] #1491 
- [ ] #1563
- [x] #1728
- [ ] #1803
- [ ] #1488 
- [ ] #2153 
- [x] #2119
- [ ] #2152 

## Check
In this jira list there are a lot of accessibility issues in terms of product and Mística component. In the following list I listed the ticket that I think should be checked in Mística side:
- [Card & Lists](https://jira.tid.es/browse/NOV-234080)
- [Password Field](https://jira.tid.es/browse/NOV-234072)
- [Password Field - Navigation](https://jira.tid.es/browse/NOV-234066)
- [Password Field - Eye icon](https://jira.tid.es/browse/NOV-234011)
- [Password Field - Eye selected](https://jira.tid.es/browse/NOV-234010)
- [Edition state in Fields](https://jira.tid.es/browse/NOV-234064)
- [Fields navigation](https://jira.tid.es/browse/NOV-234062)
- [Autocomplete](https://jira.tid.es/browse/NOV-234034)
- [Fields](https://jira.tid.es/browse/NOV-233835)
- [Loading Bar / Progress Bar](https://jira.tid.es/browse/NOV-233829)
